doc/api/tls.md

```diff
@@ -115,6 +115,29 @@ SNI (Server Name Indication) are TLS handshake extensions:
 * SNI - Allows the use of one TLS server for multiple hostnames with different
   SSL certificates.
 
+*Note*: Use of ALPN is recommended over NPN. The NPN extension has never been
+formally defined or documented and generally not recommended for use.
```

Comment:
NPN was removed in https://github.com/nodejs/node/pull/19403

Reply:
No, it can be removed.